tests/cypress/integration/issue_1785_propagation_latest_frame.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check propagation work from the latest frame', () => {
+
+    const issueId = '1785'
+    const labelName = `Issue ${issueId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const images = [`image_${issueId}_1.png`,
+                    `image_${issueId}_2.png`,
+                    `image_${issueId}_3.png`]
+    const width = '800'
+    const height = '800'
+    const posX=10
+    const posY=10
+    const color = 'gray'
+    const archiveName = `images_issue_${issueId}.zip`
+    const archivePath = `cypress/fixtures/${archiveName}`
+    const imagesFolder = `cypress/fixtures/image_issue_${issueId}`
+    const lastElement = images[images.length - 1];
+
+    before(() => {
+        cy.visit('auth/login')
+        cy.login()
+        cy.get('[type="submit"]').click()
+        for (let img of images) {
+            cy.imageGenerator(imagesFolder, img, width, height, color, posX, posY, labelName)
+        }
+        cy.wait(2000)
+        cy.createZipArchive(imagesFolder, archivePath)
+        cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName)
+    })
+
+    describe(`Open task "${taskName}"`, () => {
+        it('The annotation task present in the list', () => {
+            cy.contains('strong', taskName)
+            .parent()
+            .parent()
+            .parent()
+            .contains('a', 'Open').click()
+        })
+        it('Open a job', () => {
+            cy.contains('a', 'Job #').click()
+            cy.url().should('include', '/jobs')
+        })
```

Comment:
We can pick out the same code in each issue to a cypress command (like ``login``)

Reply:
This changes already merged. I did a rebase and a small refactoring of the test code.